### ISSUE TITLE
Make the "Available updates" in Dashboard little bit more verbose when there are no updates

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#4313]
**Description of the need**

When there are no updates the "Available updates" block on Dashboard  looks like:

<img width="473" alt="Screen Shot 2020-02-18 at 8 11 35 PM" src="https://user-images.githubusercontent.com/56604405/74798581-5cdadb00-528b-11ea-91b6-23a62543bbbe.png">

which new users could find confusing.

**Proposed solution**

I believe when there are no updates the "Available updates" block on Dashboard should either be automatically hide itself or contain some information. For example, depending on what kind of  information user choose to display on the block's settings, it could display something like:

> Congratulations! Your website is up to date as no more updates are found for Backdrop core, modules, themes or layouts.

This feature request is a small one, that could improve user interface's usability a bit.

**Comment by BWPanda** [comment]
> I believe when there are no updates the "Available updates" block on Dashboard should either be automatically hide itself or contain some information. ...something like:
>
>> Congratulations! Your website is up to date as no more updates are found for Backdrop core, modules, themes or layouts.

I agree with this, though I have the opposite problem. When there are no available updates, it says this for each project on the site (which I think is unnecessary and distracting):

![image](https://user-images.githubusercontent.com/2385329/92530969-c3980400-f270-11ea-9fce-2bd10b5f7627.png)

I couldn't find an existing issue about why this is happening, but since this issue wants to achieve the same thing (either hiding the block or just displaying a simple text message) then I decided to post here.

**Comment by olafgrabienski** [comment]
> When there are no available updates, it says this for each project on the site

Same for me on a test site. Wondering why for @alanmels it was an empty block instead.

**Comment by stpaultim** [comment]
The block is absolutely empty when the there are no modules installed, for example in this fresh Tugboat demo site. 

![Dashboard___Backdrop](https://user-images.githubusercontent.com/3144571/94104252-3a4e0900-fdfc-11ea-82f0-0dff640b8a4b.png)

Once you add and enable the Backup and Migrate module (or any module) you get the following:

![Dashboard___Backdrop](https://user-images.githubusercontent.com/3144571/94104300-4f2a9c80-fdfc-11ea-8721-7ba84172f4c8.png)

**Comment by BWPanda** [comment]
Here's a PR: https://github.com/backdrop/backdrop/pull/3328

![image](https://user-images.githubusercontent.com/2385329/94146275-0b489d80-feb7-11ea-8cc2-62431b2245b8.png)

**Comment by klonos** [comment]
Thanks @BWPanda ...any projects that are up to date are now being omitted from the dashboard block,
and the message is shown instead 👍  ...one thing though is that I don't understand why the addition of the `dashboard-no-updates` class in the message paragraph. And why do we need the message in a `<p>` in the first place?

There are a few other issues with this block (which can be follow-up issues):

![image](https://user-images.githubusercontent.com/2423362/94181707-8c4c6880-fea8-11ea-907c-f78bfad08909.png)

- Updates for layout templates are showing the machine name instead of the human-readable label
- The "Update" link always points to `/admin/modules/update` regardless of whether the update is for a module, theme, or layout template. Ideally, we should fix ???, so this is not an issue; but till then, we should make sure that the link is appropriate to the project type of the available update.
- Alternatively to the above, why not have a single "Update" link at the bottom-right of the block, same as we do for the other Dashboard blocks?
- Can we also please show a "Last checked: x hours y min ago" message, same as we do in the "Available updates page"?

**Comment by BWPanda** [comment]
I don't think this is an issue any longer...

Here's the block with available updates:
![image](https://user-images.githubusercontent.com/2385329/155974357-07614943-56c2-4b4b-b4cf-6f33b669c61d.png)

And here it is with no updates:
![image](https://user-images.githubusercontent.com/2385329/155974451-acd0d20b-e77e-45af-88da-ba56ba630b47.png)

So I'll close this issue now. If anyone's still experiencing issues with this block, feel free to re-open.

**Comment by klonos** [comment]
For posterity, this was addressed as part of #2714. See https://github.com/backdrop/backdrop/pull/3451/files#diff-592f90d2cc1517eb0988e105cf4ceb16c280dd29fb4780922a94f833c7943a7fR167
